modules/@ergonode/ui/src/components/ParagraphWithLinks/ParagraphWithLinks.vue

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<script>
+import {
+    isEmpty,
+} from '@Core/models/objectWrapper';
+
+export default {
+    name: 'ParagraphWithLinks',
```

Comment:
I would stick to Form naming convention. Those elements are using for Forms. As it's FormParagraph, I would rename component for FormLinkParagraph - cuz it's "Hello I am simple text, please click here", where "please click here" is a link.